File: portfolio-react/src/components/Education.js
import React from 'react';
import './Education.css';

function Education({ education }) {
  return (
    <section className="section education">
      <h2 className="section-title">Education</h2>
      
      {education.map((edu, index) => (
        <article key={index} className="education-item">
          
          <div className="edu-header">
            <h3 className="institution">{edu.institution}</h3>
            <span className="date-range">{edu.dateRange}</span>
          </div>
          
          <p className="course">{edu.course}</p>
          <p className="level">{edu.level}</p>
        </article>
      ))}

    </section>
  );
}

export default Education;

File: src/model/Education.java
package model;

public class Education {
    private String institution;
    private String course;
    private String dateRange;
    private String level;

    public Education(String institution, String course, String dateRange, String level) {
        this.institution = institution;
        this.course = course;
        this.dateRange = dateRange;
        this.level = level;
    }

    public String getInstitution() { return institution; }
    public String getCourse() { return course; }
    public String getDateRange() { return dateRange; }
    public String getLevel() { return level; }
}
